kyaml/fn/framework/doc.go

```diff
@@ -104,7 +104,7 @@
 //      return items, nil
 //    }
 //
-//    p := framework.SimpleProcessor{Config: functionConfig, Fn: fn}
+//    p := framework.SimpleProcessor{Config: functionConfig, Filter: fn}
```

Comment:
It's just a comment, but I suspect you intended to set this to `Filter: kio.FilterFunc(fn)`